app/tests.py

```diff
@@ -352,3 +355,45 @@ def test_download_csv_with_info(self):
             csv_set = set(row)
         expected_csv_set = set(list(expected_upfront_dict.values()))
         self.assertEqual(csv_set, expected_csv_set)
+
+
+class FetchCaseTests(TestCase):
+    def test_fetch_cases_by_case_number(self):
+        FAKE_SF_QUERY_RESPONSES = (
```

Comment:
NIT: If you want you can create a method that works like a "factory". So instead of create this mock response "manually" to can generate multiple `records` with a function:
Example: "event factory" https://github.com/evbeda/mercury/blob/7f6fcfef7c573f9f2b01474164beb809aebe0d75/src/mercury_app/utils.py#L631
Note, they are using `Faker`, that library also could help you to define fake attributes ;)

Reply:
Seems nice, I'll try that 👍